src/components/PrivateRoute.js

```diff
@@ -2,16 +2,17 @@ import { Auth } from 'aws-amplify'
 import React,{useState, useEffect} from 'react'
 import { Redirect, Route} from 'react-router-dom'
 import Books from './Books'
-import Todo from './Todo'
 
-const PrivateRoute = ({children, ...rest}) => {
 
-    
+const PrivateRoute = ({component: Component, ...rest}) => {
     const [signInUser, setSignInUser] = useState(null)
     const [isLoading, setIsLoadingUser] = useState(true)
+    
+    // Do get current user from context api.
+
 
-    useEffect(() => {
-        let fetchUser = async() => {
```

Comment:
Use const instead of let wherever possible. This can be used with const wherever possible.